uportal-war/src/main/webapp/WEB-INF/jsp/Directory/directory.jsp

```diff
@@ -40,7 +40,7 @@
       <div class="portlet-section-body">
 
         <form action="${ formUrl }" method="POST">
-            <input name="query" value="${ fn:escapeXml(query )}"/> <input type="submit" value="<spring:message code='search.directory.submit'/>"/>
+            <label for="query"><input name="query" value="${ fn:escapeXml(query )}"/> <input type="submit" value="<spring:message code='search.directory.submit'/>"/></label>
```

Comment:
:art: Could this be broken out across several lines?

Something like:

``` jsp
<label for="query">
    Query
</label>
<input type="search" name="query" value="${ fn:escapeXml(query )} "/>
<input type="submit" value="<spring:message code='search.directory.submit'/>" />
```

Thanks!